cdap-docs/developers-manual/source/getting-started/standalone/virtual-machine.rst

```diff
@@ -20,26 +22,58 @@ The CDAP Standalone Virtual Machine is configured with the recommended settings
 
 - 4 GB of RAM
 - Ubuntu Desktop Linux
-- 10 GB of disk space
+- 40 GB of virtual disk space
 
 It has pre-installed all the software that you need to run and develop CDAP applications:
    
-- Java JDK 7 or 8 and Node.js are both installed.
-- Maven is installed and configured to work for CDAP.
-- The Standalone CDAP SDK is installed under ``/Software/cdap-sdk-``\ |literal-release|
+- The Standalone CDAP SDK is installed under ``/opt/cdap/sdk``
   and will automatically start when the virtual machine starts.
-- Both IntelliJ and Eclipse IDE are installed and will start when the virtual machine starts.
-- The Firefox web browser starts when the machine starts. Its default home page is the CDAP UI,
+- A Java JDK is installed.
+- Maven is installed and configured to work for CDAP.
+- Both IntelliJ IDEA and Eclipse IDE are installed and available through desktop links once the
+  virtual machine has started.
+- Links on the desktop are provided to the CDAP SDK, CDAP UI, and CDAP documentation.
+- The Chromium web browser is included. The default page for the CDAP UI, available through a desktop link, is
   ``http://localhost:9999``.
 
 No password is required to enter the virtual machine; however, should you need to install or
 remove software, the admin user and password are both ``cdap``.
 
 .. include:: ../dev-env.rst  
    :start-line: 7
+   :end-line: 23
+
+.. tabbed-parsed-literal::
+   :tabs: "Linux Virtual Machine"
+   :independent:
+
+   $ mvn archetype:generate \
+       -DarchetypeGroupId=co.cask.cdap \
+       -DarchetypeArtifactId=cdap-app-archetype \
+       -DarchetypeVersion=\ |release|
+
+.. include:: ../dev-env.rst  
+   :start-line: 30
+
+Starting and Stopping Standalone CDAP
+============================================
+
+Use the ``cdap.sh`` script to start and stop the Standalone CDAP:
+
+.. tabbed-parsed-literal::
+   :tabs: "Linux Virtual Machine"
+   :independent:
+
+   $ cd /opt/cdap/sdk
+   $ ./bin/cdap.sh start
+     . . .
+   $ ./bin/cdap.sh stop
```

Comment:
I tried this on Oracle VirtualBox, using CDAP 3.4.1. I didn't need `sudo` to stop or start cdap. Sure this is required?